src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/casedocuments/CorrespondenceHandler.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.casedocuments;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class CorrespondenceHandler extends PartyDocumentHandler {
+
+    public CorrespondenceHandler(String collectionName, String party, ObjectMapper mapper) {
+        super(collectionName, party, mapper);
+    }
+
+    @Override
+    protected boolean isDocumentTypeValid(String caseDocumentType) {
+        return caseDocumentType.equals("Offers")
```

Comment:
I tend to prefer inverting the fields when doing string equals to avoid NPE and possibly in this case this should be an enum?